src/main/java/com/trashit/back/controllers/UserController.java

```diff
@@ -49,26 +55,36 @@ public User getUser(@PathVariable Long id) {
     }
 
     @PostMapping
-    public User createUser(@RequestBody User newUser) {
+    public User createUser(@ModelAttribute User newUser) {
+
         User user = new User();
         user.setFirstname(newUser.getFirstname());
         user.setLastname(newUser.getLastname());
         user.setSurname((newUser.getSurname() == null)
                 ? newUser.getFirstname() + " " + newUser.getLastname()
                 : newUser.getSurname());
         user.setEmail(newUser.getEmail());
-        user.setPassword(newUser.getPassword());
+        user.setPassword(BCrypt.hashPassword(newUser.getPassword()));
         user.setStudentId(newUser.getStudentId());
-        user.setBirthday(newUser.getBirthday());
+        user.setBirthday( newUser.getBirthday()  );
         user.setPicture(newUser.getPicture());
         user.setRole(roleRepository.findRoleByName(Roles.USER));
         user.setAllowNotification(newUser.isAllowNotification());
         user.setAllowLocalisation(newUser.isAllowLocalisation());
         user.setAllowPrivateReward(newUser.isAllowPrivateReward());
         user.setChartAccepted(newUser.isChartAccepted());
+
```

Comment:
ajoute la `creationDate`
```suggestion
user.setCreatedAt(new Date());
```